code/modules/mob/living/silicon/robot/robot_modules/station.dm

```diff
@@ -497,7 +497,7 @@ var/global/list/robot_modules = list(
 		F.icon_state = "flash"
 	else if(F.times_used)
 		F.times_used--
-	var/obj/item/gun/energy/taser/mounted/cyborg/T = locate() in src.modules
+	var/obj/item/gun/energy/robotic/taser/T = locate() in src.modules
```

Comment:
As tasers can be removed, I heavily recommend a sanity check below as well in case no taser is found.

Reply:
thanks^^